### PR TITLE
Update envoy to 6e5facc (September 27, 2021)

### PR DESCRIPTION
**Pull request by dubious90**
No changes required.
- No changes to .bashrc, .bazelversion, or run_envoy_docker.sh.
- All cc files built, and all tests passed without code changes.

### CODE REVIEW

File: bazel/repositories.bzl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-ENVOY_COMMIT = "037064f27e96bf26d403ff7798aafb67db6d6030"  # Sep 20, 2021
-ENVOY_SHA = "187491b0e75943fa98620c969ed321ce4a9643602a5b0a4aeb99f43593512632"
+ENVOY_COMMIT = "6e5facc3a9c66edff6ba2cff8a684aa5747671e5"  # Sep 27, 2021
+ENVOY_SHA = ""
```
Comment (mum4k) [inline bazel/repositories.bzl:4]: Can we put the SHA value here?

**Review by mum4k — CHANGES_REQUESTED** [review]
Just to confirm, did we verify no changes are needed in either of `.bazelrc`, `.bazelversion` or `run_envoy_docker.sh`?

File: bazel/repositories.bzl (hunk 1 above)
Comment (dubious90) [reply bazel/repositories.bzl]: Done.  Sorry, when you had reviewed this, I had meant to put the draft label on it

Comment (dubious90) [reply bazel/repositories.bzl]: I'm still looking into the coverage failure

**Comment by dubious90** [comment]
> Just to confirm, did we verify no changes are needed in either of `.bazelrc`, `.bazelversion` or `run_envoy_docker.sh`?

Yes, no changes to any of those files.

**Review by mum4k — APPROVED** [review]

**Pull request merged**